core/App/utils/helpers.ts

```diff
@@ -29,6 +31,54 @@ export function hashToRGBA(i: number) {
   return '#' + '00000'.substring(0, 6 - colour.length) + colour
 }
 
+export function formatTime(time: Date, params?: { long?: boolean; format?: string }): string {
```

Comment:
Question: Do locales not work in moment.js?

Reply:
no, it seems locales don't do anything in react-native period. They don't change the translation and they don't change the date format

Reply:
😢